### DISCUSSION TITLE
Configuration Templates

### DISCUSSION THREAD

**Discussion by kkebo** [kkebo/DNSecure#23] [Ideas]
Currently, DNSecure has the examples of DNS services by default, e.g. Google Public DNS, Cloudflare 1.1.1.1, and Quad9.

However, it has the following problems:

Once you change server settings, you can't see a new default list even if you update the app that includes a new one.
If the default list is too long, the app becomes less useful.
Related to #10 (comment)
To solve them, I plan to treat those DNS services as templates and add "Create from Templates" into the context menu of the plus button so that I can leave the default list empty.

Original: #13

**Comment by nitrohorse** [comment]
Hi, thanks for developing this app; it’s really useful! Please feel free to use any premade configurations I created at https://gitlab.com/nitrohorse/ios14-encrypted-dns-mobileconfigs/-/tree/master/mobileconfig. Would be neat to eventually have that list of providers predefined in the app for even more convenience for users but of course UX and maintainability need to be considered.

**Comment by kkebo** [comment]
Resolved by #84.
